packages/module/src/CullingInfo/CullingInfo.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { ExclamationCircleIcon, ExclamationTriangleIcon } from '@patternfly/react-icons';
+import { Tooltip, TooltipProps } from '@patternfly/react-core';
+import clsx from 'clsx';
+import { createUseStyles } from 'react-jss';
+import { CullingDate, CullingInfo, calculateTooltip } from './CullingInfoUtils';
+
+export type Render = (config: { msg: string }) => React.ReactElement<any, any> | null;
+
+const useStyles = createUseStyles({
+  inventoryCullingWarning: {
+    color: 'var(--pf-v6-global--warning-color--200)',
```

Comment:
In general, we should avoid using tokens which end with a number - those are called base tokens. Instead we should always be using semantic tokens which will adjust automatically in dark theme and be more context aware.

In this case, I believe the correct variable is `--pf-t--global--icon--color--status--warning--default`

And for the danger variant, it should be `--pf-t--global--icon--color--status--danger--default`

if those colors look incorrect, we could consult @kaylachumley for different tokens